osu.Desktop/Program.cs

```diff
@@ -34,19 +38,39 @@ public static int Main(string[] args)
                 }
                 else
                 {
-                    switch (args.FirstOrDefault() ?? string.Empty)
-                    {
-                        case "--tests":
-                            host.Run(new OsuTestBrowser());
-                            break;
-                        default:
-                            host.Run(new OsuGameDesktop(args));

```

Comment:
did you check what this does?

Reply:
Launch TestBrowser or Game?

Reply:
Look at what that parameter does

Reply:
I don't know what your mean for.
Please, tell me detail

Reply:
Do you know what the parameter `args` does?

Reply:
```args``` is argument that fill launch options

Example that commendline
```
osu!.exe --tests 
```
And args insert  "--tests" in array

I hope this answers.

Reply:
https://github.com/ppy/osu/blob/master/osu.Game/OsuGame.cs#L103

Reply:
Oh, I see.
That's mean my PR is useless? :(

Reply:
You, this PR is not needed.